### PR TITLE
CAS-606: CAP Navigation Bar

### PR DESCRIPTION
**Pull request by germanurrus**
Remove components and refactored a bit, now create proposal uses nav with controls (back and next ) and create community still uses old way 
Preview will be done in another PR

Desktop Version
Screen Shot 2022-10-17 at 19 34 50
Screen Shot 2022-10-17 at 19 35 16
Screen Shot 2022-10-17 at 19 35 33

Mobile: 
Screen Shot 2022-10-17 at 19 36 01
Screen Shot 2022-10-17 at 19 36 44
Screen Shot 2022-10-17 at 19 36 48

### CODE REVIEW

File: frontend/packages/client/src/components/StepByStep/LeftPanel.js [hunk 1]
```diff
@@ -68,26 +66,6 @@ export default function LeftPanel({
...
-        <div className={classNamesWrapper}>
-          <SubmitButton
-            formId={formId}
-            disabled={!isStepValid}
-            onSubmit={onSubmit}
-            label={finalLabel}
-            isSubmitting={isSubmitting}
-          />
-        </div>
-      )}
```
Comment (germanurrus) [inline frontend/packages/client/src/components/StepByStep/LeftPanel.js:90]: submit and next buttons are not shown any more on left panel

File: frontend/packages/client/src/components/StepByStep/NavStepByStep.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
+import NavButton from './NavButton';
+
+const PossitionWrapper = ({
```
Comment (jbluks) [inline frontend/packages/client/src/components/StepByStep/NavStepByStep.js:3]: spelling mistake should be Position

File: frontend/packages/client/src/components/StepByStep/NavStepByStep.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  isTopPossition,
```
Comment (jbluks) [inline frontend/packages/client/src/components/StepByStep/NavStepByStep.js:4]: Same here

File: frontend/packages/client/src/components/StepByStep/NavStepByStep.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  showSubmitOrNext,
+  formId,
+  finalLabel,
+  onClickPreview = () => {},
+  onSubmit,
+  isSubmitting,
+  isPreviewModeVisible,
+  position,
+}) {
+  const isTopPossition = position === 'top';
```
Comment (jbluks) [inline frontend/packages/client/src/components/StepByStep/NavStepByStep.js:50]: Same

**Review by jbluks — APPROVED** [review]
A few spelling corrections

**Pull request merged**